### PR TITLE
MC-13607: added docs for list site api call

### PR DESCRIPTION
**Pull request by RGitB**
#### Changes made

-  Added a new document for listing SP// sites

### CODE REVIEW

File: source/includes/stackpath/_sites.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+Optional | &nbsp;
+------ | -----------
+`status`<br/>*string* | Setting the status parameter to **ACTIVE** will return only the active **stackpath sites**.
+
+Attributes | &nbsp;
+------- | -----------
+`id`<br/>*string* | A sites's unique identifier. 
+`stackId`<br/>*string* | The ID of the stack that a site belongs to.
+`domain`<br/>*string* | The domain of the site.
+`status`<br/>*string* | The status of the site. It can be either `READY`, `SUSPENDED`, or `DISABLED`.
```
Comment (RGitB) [inline source/includes/stackpath/_sites.md]: I couldn't find the supported statuses for sites in SP//. These are the ones sites usually have. Any suggestions?

File: source/includes/stackpath/_sites.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
+## Sites
+
+Stackpath allows you to create and manage your CDN, WAF, and/or Serverless Scripting Delivery sites.
```
Comment (patrickspensieri) [inline source/includes/stackpath/_sites.md]: How about just "Create and manage your CDN, WAF, and/or Serverless Scripting Delivery sites."

File: source/includes/stackpath/_sites.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+`status`<br/>*string* | Setting the status parameter to **ACTIVE** will return only the active **stackpath sites**.
+
+Attributes | &nbsp;
+------- | -----------
+`id`<br/>*string* | A sites's unique identifier. 
+`stackId`<br/>*string* | The ID of the stack that a site belongs to.
+`domain`<br/>*string* | The domain of the site.
+`status`<br/>*string* | The status of the site. It can be either `READY`, `SUSPENDED`, or `DISABLED`.
+`createdAt`<br/>*string* | Creation timestamp of the site.
+`updatedAt`<br/>*string* | Last updation timestamp of the site.
```
Comment (patrickspensieri) [inline source/includes/stackpath/_sites.md]: I don't think "updation" makes sense here, maybe "The date on which the site was last updated" or "Timestamp of when the site was last updated" (for both list and retrieve)

File: source/includes/stackpath/_sites.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+}
+```
+
+<code>GET /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/sites</code>
+
+Retrieve a list of all sites in a given [environment](#administration-environments).
+
+Optional | &nbsp;
+------ | -----------
+`status`<br/>*string* | Setting the status parameter to **ACTIVE** will return only the active **stackpath sites**.
```
Comment (y0geshdev) [inline source/includes/stackpath/_sites.md]: Should we include what other possible values this `status` can have?

File: source/includes/stackpath/_sites.md [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+
+Attributes | &nbsp;
+------- | -----------
+`id`<br/>*string* | A sites's unique identifier. 
+`stackId`<br/>*string* | The ID of the stack that a site belongs to.
+`domain`<br/>*string* | The domain of the site.
+`status`<br/>*string* | The status of the site. It can be either `READY`, `SUSPENDED`, or `DISABLED`.
+`createdAt`<br/>*string* | Creation timestamp of the site.
```
Comment (y0geshdev) [inline source/includes/stackpath/_sites.md]: Similar to @patrickspensieri below the comment and also to keep it consistent - How about `The date on which this site was created.` ?

File: source/includes/stackpath/_sites.md (hunk 2 above)
Comment (RGitB) [reply source/includes/stackpath/_sites.md]: 8e19fea

File: source/includes/stackpath/_sites.md (hunk 3 above)
Comment (RGitB) [reply source/includes/stackpath/_sites.md]: Thanks, fixed it. 8e19fea

File: source/includes/stackpath/_sites.md (hunk 4 above)
Comment (RGitB) [reply source/includes/stackpath/_sites.md]: 8e19fea

File: source/includes/stackpath/_sites.md (hunk 5 above)
Comment (RGitB) [reply source/includes/stackpath/_sites.md]: 8e19fea

**Review by y0geshdev — APPROVED** [review]
LGTM.
Thanks for the changes.

File: source/includes/stackpath/_sites.md (hunk 1 above)
Comment (patrickspensieri) [reply source/includes/stackpath/_sites.md]: I didn't find anything either in the docs. Let's update if we find new statuses while working on sites.

**Review by patrickspensieri — APPROVED** [review]
Thanks for making the changes ✔️

File: source/includes/stackpath/_sites.md (hunk 1 above)
Comment (RGitB) [reply source/includes/stackpath/_sites.md]: Sounds good. @y0geshchaudhary suggested creating and deleting new sites and querying for status right away. I did that and found statuses to be `ACTIVE`, `PENDING`, or `PROVISIONING`. So, updated to those for now

**Pull request merged**